src/vs/workbench/services/configurationResolver/common/variableResolver.ts

```diff
@@ -278,7 +278,8 @@ export class AbstractVariableResolverService implements IConfigurationResolverSe
 						}
 						const dirname = paths.dirname(getFilePath());
 						if (folderUri || argument) {
-							return paths.relative(this.fsPath(getFolderUri()), dirname);
+							const relative = paths.relative(this.fsPath(getFolderUri()), dirname);
+							return relative.length === 0 ? '.' : relative;
```

Comment:
We used to call `paths.normalize(relative)`, which does the conversion from empty string to '.'. Since it isn't safe to call `normalize` on a path that could come from a remote, I removed it. In this case though, we still need the '.' instead of empty string.

Reply:
There's potential for refactoring here as `fsPath` and `path.relative` should not be used on remote URIs.

But for the fix release the change is minimal and good.